corgi/core/models.py

```diff
@@ -1010,9 +1010,12 @@ def released_components(self, include: bool = True) -> "ComponentQuerySet":
         errata_relations = Q(software_build__relations__type=ProductComponentRelation.Type.ERRATA)
         if include:
             # Truthy values return only released components
-            return self.filter(errata_relations)
-        # Falsey values return only unreleased components
-        return self.exclude(errata_relations)
+            queryset = self.filter(errata_relations)
+        else:
+            # Falsey values return only unreleased components
+            queryset = self.exclude(errata_relations)
+        # Clear ordering and apply distinct() to avoid duplicates from above filter
+        return queryset.order_by().distinct()
```

Comment:
What happens when you combine release_components with manifest_components? I guess DJango is clever enough to only deduplicate once all the filter is complete? If so, think it's a good idea to add it here, otherwise I think we should add it only on the 'final' filter.